src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -45,6 +45,9 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <unordered_set>
 #include <regex>
 
+/// Given a string of the format '?blah?', will return true when compared
+/// against a string that matches appart from any characters that are '?'
+/// in the original string. Equivalent to doing a regex match on '.blah.'
```

Comment:
If it doesn't take to long I intend to remove this completely and replace its uses with `std::regex` unless anyone objects to that?

Reply:
Knock up a quick standalone benchmark before you do, as this is obviously a much simpler machine